### ISSUE TITLE
Ability to make a github release (tared file and under the releases tab)

### ISSUE THREAD

**Issue by iangregsondev** [Dev-Thought/nx-plugins#50]
It would be nice to be able to support GITHUB releases. The release would basically create a Tar file and upload it to the github releases tab.

This request was originally under a different issue but I am separating by request.

**Comment by MitkoTschimev** [comment]
We have to think about the plugin self and it is built as a helper to end up with a running instance.
I don't see that a Github release will help us there? It is just a build and not really a deployment.

**Comment by iangregsondev** [comment]
I agree - not a true deployment. Although you end up publishing the contents to github for people to see the releases / changelogs etc.

I believe there are github actions already that support this kind of thing BUT no integration with NX directly.
